code/__DEFINES/lighting.dm

```diff
@@ -32,17 +32,6 @@
 /// Set to zero to disable soft lighting. Luminosity changes then work if it's lit at all.
 #define LIGHTING_SOFT_THRESHOLD 0
 
-/// If I were you I'd leave this alone.
-#define LIGHTING_BASE_MATRIX \
-	list                     \
```

Comment:
:eyes:

Reply:
tfw <https://github.com/tgstation/tgstation/pull/58991>

Reply:
the instructions were VERY clear

Reply:
They were not me